src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -1,27 +1,27 @@
 package name.valery1707.kaitai;
 
-import org.apache.maven.plugin.MojoExecutionException;
-import org.apache.maven.plugin.logging.Log;
 import org.buildobjects.process.ExternalProcessFailureException;
 import org.buildobjects.process.ProcBuilder;
 import org.buildobjects.process.StartupException;
 import org.buildobjects.process.TimeoutException;
+import org.slf4j.Logger;
 
 import java.nio.file.Path;
 import java.util.LinkedHashSet;
 import java.util.Set;
 
 import static java.util.Collections.unmodifiableSet;
-import static name.valery1707.kaitai.MojoUtils.*;
+import static name.valery1707.kaitai.KaitaiUtils.*;
```

Comment:
\[Checkstyle\] WARNING: Using the '\.\*' form of import should be avoided \- name\.valery1707\.kaitai\.KaitaiUtils\.\*\.